starter-bots/KotlinBot/src/main/kotlin/galaxio/Main.kt

```diff
@@ -0,0 +1,86 @@
+package galaxio
+
+import com.microsoft.signalr.HubConnectionBuilder
+import com.microsoft.signalr.HubConnectionState
+import galaxio.model.GameObject
+import galaxio.model.GameState
+import galaxio.model.dto.GameStateDto
+import galaxio.service.BotService
+import java.io.EOFException
+import java.util.*
+import java.util.concurrent.TimeUnit
+import java.util.logging.LogManager
+
+object Main {
+	private fun getRunnerUrl(): String {
+		var ip = System.getenv("RUNNER_IPV4")
+		if (ip == null || ip.isBlank()) {
+			ip = "localhost"
+		}
+		if (!ip.startsWith("http://")) {
+			ip = "http://$ip"
+		}
+		return "$ip:5000/runnerhub"
+	}
+
+	@JvmStatic
+	fun main(args: Array<String>) {
+		val token = System.getenv("TOKEN") ?: UUID.randomUUID().toString()
```

Comment:
The correct System Environment variable is REGISTRATION_TOKEN. 
If the token is not correct, we cannot update the bot's score and placement.

Reply:
Am I misunderstanding something or is the Java starter bot also wrong then?
https://github.com/EntelectChallenge/2021-Galaxio/blob/b5b0f51e3fb4854fdc9d3f4104e3ec60e86b5f4e/starter-bots/JavaBot/src/main/java/Main.java#L15

Reply:
The documentation also seems to match the Java bot.
https://github.com/EntelectChallenge/2021-Galaxio/blob/develop/building-a-bot.md#21-get-your-token-from-your-environment

Reply:
The token must be the Pascal Case `Token` or alternatively `REGISTRATION_TOKEN`.

I will update the documentation to correct this, apologies. But please update here

Reply:
Thanks for the clarification. :slightly_smiling_face: 

Fixed in 57e6ad502636b30e2246b5fa8d84b255e6c5a5d4.